### ISSUE TITLE
Input binding not working with iOS Chrome autofill

### ISSUE THREAD

**Issue by AlecRust** [featurist/hyperdom#62]
If you have a form that uses Hyperdom's `binding` (which I assume listens to the input's `onchange` event) and then fill that form with iOS Chrome's autofill, the `onchange` event doesn't get fired when the input is filled. So it looks like it's filled but when you proceed to next input you can see it hasn't:

![screencast](https://giant.gfycat.com/NeatNastyJackal.gif)

Not sure this is a Hyperdom issue really, same probably happens in React/Angular etc. Possibly related Chrome issue: https://bugs.chromium.org/p/chromium/issues/detail?id=352527

Have you come across this on other Hyperdom forms when you try to use Autofill?

**Note:** I don't get this issue on macOS Chrome.

**Comment by refractalize** [comment]
Hi @AlecRust, does this happen in iOS Safari?

**Comment by AlecRust** [comment]
@refractalize no, the autofill in the standard Mobile Safari seems to fill these inputs fine.

**Comment by thom801** [comment]
@AlecRust I am running into this same issue and I think it just has to do with Chrome cause I'm not using much on front end aside from Vue.js. Did you find a solution for this?

**Comment by ElMatella** [comment]
Yep, I've got the same problem here with Vue, an issue was once opened, but the problem seems to be back

https://github.com/vuejs/vue/issues/5248
https://github.com/fergaldoyle/vue-form/issues/81
